tests/sharding/test_tester_chain_handler.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from viper import compiler
+
+from eth_tester.backends.pyevm.main import get_default_account_keys
+
+from evm.utils.address import generate_contract_address
+
+import eth_utils
+
+from evm.chains.sharding.mainchain_handler.config import (
+    GASPRICE,
+    PASSPHRASE,
+    TX_GAS,
+)
+
+from evm.chains.sharding.mainchain_handler.vmc_utils import (
+    decode_contract_call_result,
+    mk_contract_tx_obj,
+)
+
+from evm.chains.sharding.mainchain_handler.backends.tester_chain_handler import (
+    TesterChainHandler,
+)
+
+keys = get_default_account_keys()
+
+@pytest.fixture(scope='function')
```

Comment:
`'function'` scope is the default and doesn't have to be specified.  Makes for slightly cleaner looking fixtures.

```python
@pytest.fixture
def thing(...):
    ...
```

Reply:
No problem. Thank you!